### ISSUE TITLE
Use outside of foobar

### ISSUE THREAD

**Issue by stuaxo** [regorxxx/Music-Graph#2]
I don't mind foobar2000, but I spend most of my time in Linux, and don't want to us a music player via Wine.

How hard would it be to make this work outside foobar2000 ?

It would be interesting to have an API attached to this + then either run it locally, and talk to it from a whatever music player, or stick it on a NAS where your music lives and connect up a DAAP server.

**Comment by regorxxx** [comment]
This is something I already thought BUT this is only the genre-similarity part.

You would need a full fledged library manager which caches the track tags and some kind of local file access to read it.
I think it would be doable on Beets -which works on unix- and allows arbitrary plugins via python, and running this on Node.js locally. 
Most similarity-comparison code is not related to foobar, so it should be doable, since node is js.

All this, although possible, is not a priority for me right now... since it involves completely changing the actual focus on foobar and refining the code logic. I can try to make it easier for other coders, or even the node js thing during this year. But the beets part and final implementation requires help by someone else.

**Comment by stuaxo** [comment]
Yep, I think node.js sounds like the path of least resistance, then the node bit could provide some API over JSON (or maybe there is some more suitable existing API)
